examples/others/raylib_opengl_interop.c

```diff
@@ -38,6 +38,9 @@
             #include <OpenGL/gl3.h>     // OpenGL 3 library for OSX
             #include <OpenGL/gl3ext.h>  // OpenGL 3 extensions library for OSX
         #else
+            #if defined(_WIN32) && defined(_MSC_VER)
+                #define GLAD_IMPLEMENTATION  // Need self-implementation for MSVC
```

Comment:
Can you explain why the "self-implementation" is needed?

Reply:
For MSVC, there are only function declarations in glad.h such that the linker can't find the function definitions(or symbols, I'm not familiar with these terms). I need to define `GLAD_IMPLEMENTATION` to offer the function definitions.
No need to define `GLAD_IMPLEMENTATION` in Linux, MSYS and MinGW. I tested it. However, to be honest, I'm not clear about the reasons.

Reply:
Something about it doesn't seem right. I think it should not be necessary, on MSVC also, and something else is wrong.

Reply:
I also think so. I found some errors happened after these changes. `raylib_opengl_interop` does not work as expected.

Does I need to close this pull request?